CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to Grafana's Shared Workflows Repository
+
+Welcome to the Shared Workflows repository! This repository contains reusable actions designed to streamline and automate common tasks across multiple departments and external organizations. We appreciate your interest in contributing to our shared efforts.
+
+## Scope of `shared-workflows`
+
+This repository is for generic workflows and actions which can be used across multiple types of projects. For more specialised use-cases, we encourage the use of project- or area-specific shared workflow repositories.
+Examples which would fit in here:
+
+- Push a container image to a Docker registry
+- Run security scanners on built artifacts
+- Read and write objects to cloud object storage (GCS or S3)
+  And those which would be better in project- or area-specific shared workflow repositories:
+- Implement a release process specific to a subset of projects
+- Lint project(s) with a single (configurable) toolset
```

Comment:
Was misleading due to the above suggestion, but the team-specific one goes here:

```suggestion
- Lint project(s) with your team's opinionated toolset
```